model/clusters_mgmt/v1/addon_upgrade_policy_state_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an addon upgrade policy state that that is set for a cluster.
+class AddonUpgradePolicyState {
+	// State value can be 'pending', 'scheduled', 'cancelled', 'started', 'delayed',
+	// 'failed' or 'completed'.
```

Comment:
Does the metamodel have a way to enum this?

Reply:
Check out: 
ocm-api-model/model/clusters_mgmt/v1/aws_infrastructure_access_role_grant_type.model

ocm-api-model/model/clusters_mgmt/v1/aws_infrastructure_access_role_grant_state_type.model

Reply:
That is how it is, we have the enum via the type https://github.com/openshift-online/ocm-api-model/blob/main/model/clusters_mgmt/v1/UpgradePolicyStateValue.model

Reply:
My bad. All good then